docs/concepts/cluster-api.md

```diff
@@ -0,0 +1,11 @@
+
+Incidentally, Gardener influenced the Machine API in the Cluster API with its [Machine Controller Manager](https://github.com/gardener/machine-controller-manager) and was the [first to adopt it](https://github.com/kubernetes-sigs/cluster-api/commit/00b1ead264aea6f88585559056c180771cce3815), see also [joint SIG Cluster Lifecycle KubeCon talk](https://www.youtube.com/watch?v=Mtg8jygK3Hs) where @hardikdr from our Gardener team in India spoke.
+
+That means, we follow the [Cluster API](https://github.com/kubernetes-sigs/cluster-api#cluster-api) with great interest and are active members. It was completely overhauled from `v1alpha1` to `v1alpha2`. But because `v1alpha2` made too many assumptions about the bring-up of masters and was enforcing master machine operations (see [here](https://cluster-api.sigs.k8s.io/user/concepts.html#control-plane): “As of `v1alpha2`, Machine-Based is the only control plane type that Cluster API supports”), services that managed their control planes differently like GKE or Gardener couldn't adopt it (e.g. [Google only supports `v1alpha1`](https://cloud.google.com/anthos/gke/docs/on-prem/concepts/cluster-api)). In 2020 [`v1alpha3`](https://kubernetes.io/blog/2020/04/21/cluster-api-v1alpha3-delivers-new-features-and-an-improved-user-experience/) was introduced and made it possible (again) to integrate managed services like GKE or Gardener. The mapping from the Gardener API to the Cluster API is mostly syntactic.
+
+To wrap it up, while the Cluster API knows about clusters, it doesn't know about their make-up. With Gardener, we wanted to go beyond that and harmoise the make-up of the clusters themselves and make them homogenous across all supported infrastructures. Gardener can therefore deliver homogenous clusters with exactly the same configuration and behaviour on all infrastructures (see also [Gardener's coverage in the official conformance test grid](https://k8s-testgrid.appspot.com/conformance-all)).
```

Comment:
First paragraph uses `harmonize` while this paragraph uses `harmonise`. Choose either American or British english 😄